### ISSUE TITLE
Addon doesn't work anymore

### ISSUE THREAD

**Issue by jimmyeatsquaids** [piyushsoni/BetterLoopyForYouTube#1]
there's no loopy bar anymore when I go to youtube videos. this problem has been there for quite some time now, but I didn't bother asking for an update. does the addon work for you guys?

 currently using firefox 42.0, windows 7 64 bit.

**Comment by piyushsoni** [comment]
Hi,
Yes, I expect to submit a fix in the next few days. Thanks!

**Comment by worldsdream** [comment]
Thanks for the heads up. Are you planning to add it to addons.mozilla.org? So it does update automatically. Without checking github everytime for it?

**Comment by piyushsoni** [comment]
Yes, that's the main motive. Uploading it here is to make the source open, and to help me manage the changes. As soon as it's working properly, I will upload it on the official Firefox AMO.

**Comment by PippityL** [comment]
Can't wait! Was so happy after I found this here after userscripts died. Can't wait to see it on the add-on page.

**Comment by piyushsoni** [comment]
Hi all,
Can any of you try the latest version of the add-on uploaded here on GitHub? It's mostly working, just that due to some latest changes by YouTube, if you go to a video from the YouTube Home/search page, it doesn't load the Better Loopy Plugin (you have to refresh it once - after that all videos will have the plugin active). 

It's still not uploaded to Firefox AMO, please bear with me for a little while more. You can test it from here : https://github.com/piyushsoni/BetterLoopyForYouTube/raw/master/BetterLoopyForYouTube.xpi  . Like any other unsigned extension, if you haven't done it already, you might have to set **xpinstall.signatures.required** to **false** by going to about:config 

Thanks,
Piyush

**Comment by ghost** [comment]
I tested it, works, but fails to load sometimes, a refresh does the trick. Thanks!

**Comment by piyushsoni** [comment]
Thanks for testing Rijul! A fix for that is coming soon with other changes.
